block/retriever.go

```diff
@@ -41,6 +44,28 @@ func (m *Manager) syncUntilTarget(syncTarget uint64) error {
 		return nil
 	}
 
+	var stateIndex uint64
+	h := m.Store.Height()
+	err := retry.Do(
+		func() error {
+			res, err := m.SLClient.GetHeightState(h)
```

Comment:
why are we exiting on error? seems to me like we have no where to proceed if we can't get the height

Reply:
we're not exiting, this retry has attempts=0 so it will retry forever